src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,13 @@
+package lotto.domain;
+
+public class LottoNumber {
+    private final int number;
+
+    public LottoNumber(Integer number) {
```

Comment:
```suggestion
    public LottoNumber(int number) {
```

래핑 타입을 사용하면 null을 입력할 수도 있으니 아예 null 자체를 입력하지 못하게 막는게 좋겠네요